src/layouts/AppMenu/AppMenuItem.astro

```diff
@@ -0,0 +1,21 @@
+---
+import type { AppMenuFragment } from '@lib/datocms/types';
+import LinkToRecord from '@components/LinkToRecord/LinkToRecord.astro';
+
+export interface Props {
+  item: AppMenuFragment['menuItems'][0];
```

Comment:
```suggestion
  item: AppMenuFragment['menuItems'][number];
```
This does not have an impact in this case, but there might be situations where an array is interpreted as a tuple by Typescript, and if this pattern is repeated in that instance you would have incorrect types.